eng/azure-pipelines.yml

```diff
@@ -101,6 +101,28 @@ stages:
           parameters:
             enablePublishTestResults: true
 
+      - ${{ if eq(variables.officialBuild, 'false') }}:
```

Comment:
Why is it not included in the existing --integrationTest ?

Reply:
You mean why doesn't the existing ``integrationTest` imply bootstrap? Good question. First reason is that I didn't know about that flag until I started on adding the pipeline, which I did last.

I think I might be able to integrate it. Let me take a shot at it.